tests/read_ntfs.rs

```diff
@@ -168,7 +168,9 @@ fn ntfs() {
     }
     assert_eq!(4, pt_objects.comments.len());
     let rera_comment_indexes = &pt_objects.lines.get("RERA").unwrap().comment_links;
```

Comment:
These are not indexes anymore. Variable name can probably be adjusted.

```suggestion
    let rera_comment_ids = &pt_objects.lines.get("RERA").unwrap().comment_links;
```